### DISCUSSION TITLE
Reducing distance in KvSimplicity

### DISCUSSION THREAD

**Discussion by Imerion** [tsujan/Kvantum#881] [Q&A] [answered]
Hi! I have been using Kvantum on my KDE desktop for a while now and I really like it! I really appreciate rich themes with lots of detail, shadows, gradients, etc and Kvantum has several nice ones. Thanks for creating and maintaining Kvantum!

Currently I'm using KvSimplicity and I'm looking for a way to reduce the distance between the menu and the toolbars slightly. I know there is a small blue bar that appears when hovering over a menu item (a really nice touch!) but even after that there is a lot of extra space that the other themes don't have. Even after looking through the config file several times I can't seem to find a setting for this. Can I adjust this somehow?

![Distance](https://github.com/tsujan/Kvantum/assets/16926509/f2220538-d7b7-411f-8861-5abe8acb04f2)

**Comment by tsujan** [comment, marked as answer]
Only 4 pixels of the red area in your screenshot is taken by the menu-bar; the rest is reserved for the text by Qt, and some of it is related to the tool-bar below it.

Anyway, you could save the theme and add `text.margin.bottom=0` under the section `[MenuBarItem]` in `~/.config/Kvantum/KvSimplicity#/KvSimplicity#.kvconfig`. If you don't want the bottom line on mouseover, you could change `frame.bottom=2` to `frame.bottom=0` under the same section.

If you also want to make the tool-bar thinner, decrease `frame.bottom=5` under `[Toolbar]`.

A comparison with Fusion after these changes:

![menubar](https://github.com/tsujan/Kvantum/assets/981076/3597e90d-b3ec-4c1e-b4b7-75043574f9aa)

Of course, this isn't what I wanted for KvSimplicity, but Kvantum can be customized.

Everything is explained in the documentation, inside the source.

**Reply by Imerion** [reply]
Thanks! That was what I was looking for. I'm not sure what the end result will be, for now I am mostly experimenting a bit. I read the documentation but I wasn't sure, so thanks for the example!

**Reply by tsujan** [reply]
> I'm not sure what the end result will be

Please also note that the result may depend on the SVG file of the theme. That file is made based on the `kvconfig` file; so, you might lose some borders or even whole elements by setting a pixel value to zero. That's why Kvantum Manager doesn't cover such settings.

However, by changing the SVG file alongside the `kvconfig` file, you could make a new theme by starting from an existing one, as explained in the documentation...

**Reply by Imerion** [reply]
I understand, thanks! In the future it might be fun to try to make completely new themes and share - we'll see what happens. Thanks again for the help!
